### DISCUSSION TITLE
Apps fail with Unhandled exception. System.BadImageFormatException

### DISCUSSION THREAD

**Discussion by devployment** [datalust/seq-tickets#1153] [Help]
After updating (from 2020.x) to the latest Seq version, certain apps fail like this

![image](https://user-images.githubusercontent.com/7137845/110292563-baa0a900-7fed-11eb-9f3e-f3829101eec8.png)

Seq version: Seq 2021.1.5307
Seq host: Windows Server 2019 Datacenter (1809)

One example is the Slack integration https://github.com/bytenik/Seq.App.Slack

As app compatibility is mentioned on the upgrade notes I suppose that this is related directly to the app implementation itself. Any hints?

**Comment by devployment** [comment]
Weird, after restarting the Seq service the apps seem to work again. Will watch if the apps stop working again at some point.

**Reply by nblumhardt** [reply]
Very odd - thanks for the update, @devployment. Fingers crossed that you don't hit this again, but if it does show up a second time we'll dig in further 👍
